cranelift/zkasm_data/generated/counter.zkasm

```diff
@@ -8,15 +8,15 @@ function_1:
   SP + 2 => SP
   B :MSTORE(SP - 1)
   0 => A
-  :JMP(L1_1)
-L1_1:
```

Comment:
The function index X in "LX_1" was important to disambiguate labels in different functions.
Imagine the following program:
```wat
(module
 (import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))
 (func $first_counter
	(local $counter i32)
	(local.set $counter (i32.const 0))
	(block
	 (loop
		(local.set $counter
		 (i32.add
			(local.get $counter)
			(i32.const 1)))
		(br_if 1
		 (i32.eq
			(local.get $counter)
			(i32.const 10)
		 )
		)
		(br 0)
	 )
	)
	(local.get $counter)
	(i32.const 10)
	call $assert_eq)

 (func $second_counter
	(local $counter i32)
	(local.set $counter (i32.const 0))
	(block
	 (loop
		(local.set $counter
		 (i32.add
			(local.get $counter)
			(i32.const 1)))
		(br_if 1
		 (i32.eq
			(local.get $counter)
			(i32.const 10)
		 )
		)
		(br 0)
	 )
	)
	(local.get $counter)
	(i32.const 10)
	call $assert_eq)
(start $second_counter))
```

It will now generate the duplicate label `label_1` in both of these functions, which will yield an invalid zkasm.

We need to keep some label disambiguation between functions.

Reply:
Thanks for notice! Fixed